networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -0,0 +1,23 @@
+//Network Observability Operator Release Notes
+:_content-type: ASSEMBLY
+[id="network-observability-operator-release-notes"]
+= Network Observability Operator release notes
+:context: network-observability-operator-release-notes-v0
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The Network Observability Operator enables administrators to observe and analyze network traffic flows for {product-title} clusters.
+
+These release notes track the development of the Network Observability Operator in the {product-title}.
+
+For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+
+[id="network-observability-operator-release-notes-1-1"]
+== Network Observability Operator 1.1.0
+
+The following advisory is available for the Network Observability Operator 1.1.0:
+
+* link:https://access.redhat.com/errata/RHSA-2023:0786[RHSA-2023:0786 Network Observability Operator Security Advisory Update]
+
+The Network Observability Operator is now stable and the release channel is upgraded to `v1.1.0`.
```

Comment:
@skrthomas - this is different than channels "stable" and v1.0.x that shows up when NOO is installed, correct?

Reply:
@memodi Correct, in this context, it means that the Operator is tested as thoroughly as possible and is reliable. However, I do see how this could be confusing lanugage if we ask users to switch to a "stable" channel vs "v1.0.x". Stable does not currently show up, right? Its just "v1.0.x" in 1.1. Then for 1.2 there was discussion about switching to stable, but I'm unsure of what the decision is there.

Reply:
> Stable does not currently show up, right?
Yes, for 1.1 it does not. For 1.2 it's tracked here: https://issues.redhat.com/browse/NETOBSERV-853

Reply:
I think we'll make it clear by using `` around `stable` and `v1.0.x` if we do ask users to switch to stable to get updates.